llpc/context/llpcCompiler.cpp

```diff
@@ -859,6 +859,13 @@ static bool isUnrelocatableResourceMappingRootNode(const ResourceMappingNode *no
         // The code to handle a compact descriptor cannot be easily patched, so relocatable shaders assume there are
         // no compact descriptors.
         return true;
+#if (LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= 50)
```

Comment:
nit: the other code does not use parens around simple version checks, e.g.: https://github.com/GPUOpen-Drivers/llpc/pull/1559/files. Also elsewhere in this PR.

I don't think this matters much, but I'd be leanings towards keeping things consistent.

Reply:
I had cut & paste from https://github.com/GPUOpen-Drivers/llpc/blob/14ddff5b342f0578a88a10f610b948c3ff1bc907/include/vkgcDefs.h#L268.

I can change it if AMD cares.